### PR TITLE
Make code more obvious.

### PR DESCRIPTION
**Pull request by philippeitis**
```if done if white else not done```
is pretty hard to parse - "if done == white" is a lot easier to understand.

### CODE REVIEW

**Review by Exahilosys — CHANGES_REQUESTED** [review]
I agree, change it up to `value in store is white` and I'll merge. 
This avoids creating `done` and `is` is faster than `==` for bools.

**Comment by philippeitis** [comment]
Done.

**Review by Exahilosys — APPROVED** [review]
